### PR TITLE
Destination ClickHouse: remove tcp port spec parameter

### PR DESCRIPTION
**Pull request by mshustov**
## What
ClickHouse configuration in Airbyte requires both HTTP and Native port to be specified. 
The HTTP port is used for JDBC driver to communicate with ClickHouse, the native port is used by dbt-clickhouse during transformation step. 
The native port is not a requirement for DBT-clickhouse anymore since the last the last versions of dbt-clickhouse support both HTTP and native protocols. We want to switch to HTTP by default everywhere since it reduces the technical complexity of the integration.

## How
This PR removes requirements for Native port to be provided to reduce the configuration size and improve UX. I removed all the mentions of TCP port in the codebase introduced in https://github.com/airbytehq/airbyte/pull/9340 and https://github.com/airbytehq/airbyte/pull/14783

## Recommended reading order
1. `*.yml` files
2. the rest

## 🚨 User Impact 🚨
The number of required fields in ClickHouse configuration was reduced. I didn't manage to run Airbyte locally (neither master branch nor the current one) to take a screenshot. In short, it should remove `native port` field:

`SUB_BUILD=PLATFORM ./gradlew build`  fails with
```
DockerProcessFactoryTest > testEnvMapSet() FAILED
    io.airbyte.workers.exception.WorkerException: Could not find image: busybox
```

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

File: airbyte-config/init/src/main/resources/seed/destination_definitions.yaml [hunk 1]
```diff
@@ -70,6 +70,7 @@
   dockerImageTag: 0.1.12
   documentationUrl: https://docs.airbyte.io/integrations/destinations/clickhouse
   releaseStage: alpha
+  icon: cliskhouse.svg
```
Comment (marcosmarxm) [inline airbyte-config/init/src/main/resources/seed/destination_definitions.yaml:73]: typo

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by marcosmarxm** [comment]
Thanks for the contribution @mshustov someone from the team will review it later this week or next one.

**Review by grishick — APPROVED** [review]
LGTM.
@mshustov  There are several small things that need to happen before this PR can be published and merged:

- update version in Dockerfile in destination-clickhouse and destination-clickhouse-strict-encrypt to 0.2.0
- add a line to the change log in clickhouse.md
- remove changes from YAML files in seed folder (these will be updated by publish command)

**Comment by marcosmarxm** [comment]
Thanks for the review @grishick! @mshustov let me know if you need help with Greg's comment.

**Comment by mshustov** [comment]
@marcosmarxm @grishick done!

**Comment by grishick** [comment]
I think there is one more change that is needed here to fix the failing unit test. I'll am testing the theory here: https://github.com/airbytehq/airbyte/pull/17483

**Comment by grishick** [comment]
I ended up adding two more changes to fix the normalization tests, publishing the new versions and merging these changes in this [pr](https://github.com/airbytehq/airbyte/pull/17483). I

**Pull request closed without merging**